app/uk/gov/hmrc/nisp/views/page_not_found_template.scala.html

```diff
@@ -0,0 +1,41 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.partials.FormPartialRetriever
+@import uk.gov.hmrc.nisp.views.viewParams.MainTemplateParams
+@import play.api.Application
+@import uk.gov.hmrc.renderer.TemplateRenderer
+
+
+@()(implicit lang: Lang, request: Request[_], messages: Messages, application: Application, formPartialRetriever: FormPartialRetriever, templateRenderer: TemplateRenderer)
+
+@anchor(text: String, url: String) = {<a href="@url" target="_blank">@text</a>}
+
+
+@unauthenticatedMain(MainTemplateParams(pageTitle = Some(Messages("global.page.not.found.error.title")))) {
```

Comment:
Is the signout link gonna be done in a different ticket? The design pattern is we need signout if they are signed in regardless if we can. 404 fits the bill for that.

Reply:
Not sure what you mean here.

Reply:
If it's whether the signout link is appearing. I can see that is on the page from testing.

Reply:
But does it display on the not found page if you are not signed in? ie. no session hit nisp 404 page

Reply:
![image](https://user-images.githubusercontent.com/33066342/71261769-1d008180-2336-11ea-9b3f-614a8128180c.png)

Reply:
Hang on Think I get you now.

Reply:
Yeah it does

Reply:
So the frontend devs have been going through and fixing this. Do we know if theres a ticket to look at this for NISP @Muhammad-Ali-Ahmad. If so, it can be looked at elsewhere, if not we need to either fix it or raise a subsequent ticket.

Reply:
@danewatts I don't think we have a fix for this. I think we have just been enabling it on page not found pages because its better to have it then not to have it. From what I have seen and remember, page not found doesn't have auth request so it is not able to determine if the user is signed in or not...

Reply:
@thirlz95 can you confirm my understanding of this standard please?